pvlib/inverter.py

```diff
@@ -306,3 +315,123 @@ def pvwatts(pdc, pdc0, eta_inv_nom=0.96, eta_inv_ref=0.9637):
     power_ac = np.maximum(0, power_ac)     # GH 541
 
     return power_ac
+
+
+def fit_sandia(curves, p_ac_0, p_nt):
+    r'''
+    Determine parameters for the Sandia inverter model from efficiency
+    curves.
+
+    Parameters
+    ----------
+    curves : DataFrame
+        Columns must be ``'fraction_of_rated_power'``, ``'dc_voltage_level'``,
```

Comment:
Is `fraction_of_rated_power` actually required?  I don't see it getting used anywhere.

Reply:
The values are not used explicitly, but they must index the efficiency curves and be repeated for each dc_voltage_level. If working from a datasheet to prepare the dataframe for input to this function, `ac_power` needs to be calculated from `fraction_of_rated_power` which is typically used in the datasheet table. I think it would be confusing to describe the needed input without using `fraction_of_rated_power`.